src/components/Header/HeaderUserMenu/HeaderUserMenu.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+
+class HeaderUserMenu extends Component {
+
+  render() {
+    return (
+      <div>
+        <span>user avatar text</span>
+        <div>
+          <span>{this.props.user.Email}</span>
+        </div>
+      </div>
+    );
+  }
+
+};
```

Comment:
```suggestion
}
```

This curly brace is related to the class definition, right?

Reply:
💤 💤 💤